code/datums/trading/illegal.dm

```diff
@@ -0,0 +1,55 @@
+/datum/trader/ship/illegal
+	name = "Unknown"
+	origin = "Unregistered Vessel"
+	typical_duration = 10
+
+/datum/trader/ship/illegal/get_item_value(var/trading_num)
+	. = ..()
+	. *= 1.5
+
+/datum/trader/ship/illegal/gunshop
+	speech = list(TRADER_HAIL_GENERIC    = "Welcome, have a look at our armaments, just be quick.",
+				TRADER_HAIL_DENY         = "Our ship doesn't sell anything, you might've mistaken us for someone else.",
+
+				TRADER_TRADE_COMPLETE    = "Nice having business with you.",
+				TRADER_NO_BLACKLISTED   = "Not interested in that one.",
+				TRADER_NO_GOODS    = "Cash for guns, thats the deal.",
+				TRADER_NOT_ENOUGH  = "Looks you need money, more of it.",
+				TRADER_HOW_MUCH          = "That'd cost you VALUE.",
```

Comment:
I believe this is how all the other traders do it, otherwise it's just the number.
```suggestion
				TRADER_HOW_MUCH          = "That'd cost you VALUE CURRENCY.",
```